client/my-sites/checkout/checkout/index.jsx

```diff
@@ -718,7 +716,6 @@ export class Checkout extends React.Component {
 				products={ productsList }
 				selectedSite={ selectedSite }
 				setHeaderText={ setHeaderText }
-				showGuaranteeSeal={ showGuaranteeSeal }
 				isJetpackNotAtomic={ isJetpackNotAtomic }
```

Comment:
We don't need `isJetpackNotAtomic ` prop too(added in #35404)

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/35728/commits/4d27869d6849e927e495261f64f6d0667da649b8